examples/asr/conf/carnelinet/carnelinet_384.yaml

```diff
@@ -0,0 +1,259 @@
+#
+# The changes between models is captured in the following table. The rest of the parameters is the same across all models.
+# +-----------------+---------+---------+------------+----------------+------------+
+# | Model           | filters | epilog  | activation | weight         | time_masks |
+# |                 |         | filters |            | init_mode      |            |
+# +-----------------+---------+---------+------------+----------------+------------+
+# | CarneliNet-256  | 256     | 640     | Swish      | tds_uniform    | 2          |
+# +-----------------+---------+---------+------------+----------------+------------+
+# | CarneliNet-384  | 384     | 640     | Swish      | tds_uniform    | 2          |
+# +-----------------+---------+---------+------------+----------------+------------+
+# | CarneliNet-512  | 512     | 640     | Swish      | tds_uniform    | 10         |
+# +-----------------+---------+---------+------------+----------------+------------+
+# | CarneliNet-768  | 768     | 1024    | ReLu       | xavier_uniform | 10         |
+# +-----------------+---------+---------+------------+----------------+------------+
+# | CarneliNet-1024 | 1024    | 1024    | ReLu       | xavier_uniform | 10         |
+# +-----------------+---------+---------+------------+----------------+------------+
+name: &name "CarneliNet-384-8x-Stride"
+
+model:
+  sample_rate: &sample_rate 16000
+
+  train_ds:
+    manifest_filepath: ???
+    sample_rate: 16000
+    batch_size: 32
+    trim_silence: false
+    use_start_end_token: false
+    max_duration: 16.7
+    shuffle: true
+    is_tarred: false
+    tarred_audio_filepaths: null
+
+  validation_ds:
+    manifest_filepath: ???
+    sample_rate: 16000
+    batch_size: 32
+    shuffle: false
+    use_start_end_token: false
+
+  test_ds:
+    manifest_filepath: null
+    sample_rate: 16000
+    batch_size: 32
+    shuffle: false
+    use_start_end_token: false
+
+  model_defaults:
+    repeat: 5
+    dropout: 0.1
+    separable: true
+    se: true
+    se_context_size: -1
+    se_repeat: false
+    kernel_size: 11
+
+  tokenizer:
+    dir: ???  # path to directory which contains either tokenizer.model (bpe) or vocab.txt (for wpe)
+    type: ???  # Can be either bpe or wpe
+
+  preprocessor:
+    _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
+    sample_rate: *sample_rate
+    normalize: "per_feature"
+    window_size: 0.025
+    window_stride: 0.01
+    window: "hann"
+    features: &n_mels 80
+    n_fft: 512
+    frame_splicing: 1
+    dither: 0.00001
+    pad_to: 16
+    stft_conv: false
+
+  spec_augment:
+    _target_: nemo.collections.asr.modules.SpectrogramAugmentation
+    freq_masks: 2
+    time_masks: 2
+    freq_width: 27
+    time_width: 0.05
+
+  encoder:
+    _target_: nemo.collections.asr.modules.ParallelConvASREncoder
+    feat_in: *n_mels
+    activation: swish
+    init_mode: tds_uniform
+    conv_mask: true
+
+    jasper:
+      - filters: 384
```

Comment:
Can you make filters a model_defaults variable too ?

Reply:
Done.